CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 ### Changes
 
+- [#302] (https://github.com/clojure-emacs/clj-refactor.el/issues/302) `ad` now understands defs using prismatic/schema.
```

Comment:
possibly worth mentioning that it understands all def like macros/things now from libraries referred with an alias.